internal/manifests/queryfrontend/query_frontend_test.go

```diff
@@ -0,0 +1,40 @@
+package queryfrontend
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/os-observability/tempo-operator/apis/tempo/v1alpha1"
+)
+
+func TestBuildQueryFrontend(t *testing.T) {
```

Comment:
Could be this test aligned with other tests to assert on the entire object spec?

Reply:
It could.  The problem with asserting on the entire spec is that it does make it more difficult to find the cause when something fails.

Reply:
Please make this test similar like to others - assert full spec